catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/mappings/CswRecordMapperFilterVisitor.java

```diff
@@ -272,6 +295,19 @@ public Object visit(PropertyIsBetween filter, Object extraData) {
 
     @Override
     public Object visit(PropertyIsEqualTo filter, Object extraData) {
+        FilterFactory2 factory = getFactory(extraData);
+
+        if (filter.getExpression1() instanceof Function) {
```

Comment:
I should have been clearer. Not new tech debt(we didn't support functions in any visit method other than EqualTo in the first place). I am just not fully implementing Functions throughout since it doesn't make sense yet since Proximity and DivisibleBy only work with PropertyIsEqualTo Filters anyway. If we add a function that makes sense for PropertyIsGreater or PropertyIsBetween at that point we would have to plumb functions all the way through for those filter types.